dockstore-webservice/src/main/java/io/dockstore/webservice/resources/TokenResource.java

```diff
@@ -167,9 +170,17 @@ public TokenResource(TokenDAO tokenDAO, UserDAO enduserDAO, DeletedUsernameDAO d
         this.googleClientID = configuration.getGoogleClientID();
         this.googleClientSecret = configuration.getGoogleClientSecret();
         this.orcidClientID = configuration.getOrcidClientID();
+        this.orcidScope = configuration.getUiConfig().getOrcidScope();
         this.orcidClientSecret = configuration.getOrcidClientSecret();
         this.client = client;
         this.cachingAuthenticator = cachingAuthenticator;
+        try {
+            URL orcidAuthUrl = new URL(configuration.getUiConfig().getOrcidAuthUrl());
+            // orcidUrl should be something like "https://sandbox.orcid.org/" or "https://orcid.org/"
```

Comment:
Suggest also showing an example string it is transforming from in the comment (your comment has what it transforms to)

Reply:
it's the OrcidAuthURL, you can see it in the partialDockstoreTest.yml